packages/kit/test/apps/basics/src/routes/actions/enhance/+page.svelte

```diff
@@ -37,7 +37,9 @@
 	use:enhance={({ controller }) => {
 		previous?.abort();
 		previous = controller;
-		return () => count++;
+		return () => {
+			count++;
+		};
```

Comment:
Is there a reason for this :laughing:

Reply:
Yes, this was intentional.

The return value from the function was throwing an error when running the tests. The return type is expected to be `void`, not a number.

It _should_ have been throwing an error before as well, though I will have to re-run and test that.

Reply:
Ahh gotcha, yeah, that makes sense -- I'll get it patched up and we can ship the PR. As I was investigating, I found that this type issue was actually causing a (probably inconsequential) bug where we weren't awaiting the promise returned by `fallback_callback` -- that's fixed now

Reply:
That was actually something I was going to comment asking a question about earlier -- whether or not we should be awaiting that `callback` method! I figured that I would at least keep the scope of the original PR limited, but I'm glad it surfaced that fix as well. Thanks for the quick turnaround on this ⚡